site/installation/README.md

```diff
@@ -108,6 +108,6 @@ kpt help
 [gcr.io/kpt-dev/kpt-gcloud]: https://console.cloud.google.com/gcr/images/kpt-dev/GLOBAL/kpt-gcloud?gcrImageListsize=30
 [cloud-sdk]: https://github.com/GoogleCloudPlatform/cloud-sdk-docker
 
-[linux]: https://storage.googleapis.com/kpt-dev/latest/linux_amd64/kpt
-[darwin]: https://storage.googleapis.com/kpt-dev/latest/darwin_amd64/kpt
-[windows]: https://storage.googleapis.com/kpt-dev/latest/windows_amd64/kpt.exe
+[linux]: https://github.com/GoogleContainerTools/kpt/releases/latest/download/kpt_linux_amd64
```

Comment:
How about other channels besides binary download (gcloud, homebrew, docker, source)?

Reply:
Do we want the page to reflect how to get the version of the kpt that is currently on the `next` branch, or do we want this to reflect how users will do it for kpt v1? I assume the former, but the process will be somewhat different than for kpt v1. For example, I don't expect we will make kpt from the `next` branch available on homebrew or gcloud until we actually have something very close to complete.

Reply:
So if some channels are not being updated yet (e.g. gcloud or homebrew), please file issue for them to follow up.

Reply:
I'm adding visible TODO to docs so fishfooders don't file duplicate bugs. We'll remove these TODOs in m3 or m4.